security-rules.bolt

```diff
@@ -137,12 +141,12 @@ path /landsraad/votingRights {
 
 path /landsraad/questions {
   read() { isSignedIn() }
-  write() { isSwiss() }
+  write() { isSignedIn() }
```

Comment:
When you allow normal users (not admins) edit something, security rules must be more detailed. See how it's done for actions.

Reply:
Yeah I noticed yesterday, I will change it.

Reply:
This is still an open question (last one) as I could not find any way how to restrict access depending on delegateId because the structure of the `landsraad/questions` does not include delegateId at all.

Reply:
Ok the security rules should not be improved.